src/ray/protobuf/gcs.proto

```diff
@@ -407,6 +407,8 @@ message ObjectLocationInfo {
   // For objects that have been spilled to external storage, the URL from which
   // they can be retrieved.
   string spilled_url = 3;
+  // The node id that spills the object to the external storage.
```

Comment:
Can you use "disk" instead of "external storage" for all such comments? To make it clear that this is a specific case of spilling where only the original node can restore.

Reply:
Actually, I prefer to keep the consistent implementation on every external storage, meaning objects are spilled only on a node that its owner exists & restored from that node. 

Your suggestion here means when we don't use file spilling, we can restore it from every node right?